src/consts.js

```diff
@@ -26,6 +26,12 @@ module.exports = {
         dockEventHidden: "dock.event.hidden",
         dockEventIconSet: "dock.event.icon.set",
         dockEventShown: "dock.event.shown",
+        globalShortcutCmdRegister: "global.shortcut.cmd.register",
+        globalShortcutCmdIsRegistered: "global.shortcut.cmd.is.register",
+        globalShortcutCmdUnregister: "global.shortcut.cmd.unregister",
+        globalShortcutCmdUnregisterAll: "global.shortcut.cmd.unregister.all",
+        globalShortcutEventProcessFinished: "global.shortcut.event.process.finished",
```

Comment:
Could you remove this event and instead use:

- `global.shortcuts.event.registered`
- `global.shortcuts.event.is.registered`
- `global.shortcuts.event.unregistered`
- `global.shortcuts.event.unregistered.all`